packages/ndk/lib/domain_layer/usecases/zaps/zap_request.dart

```diff
@@ -2,10 +2,10 @@ import '../../entities/nip_01_event.dart';
 
 /// Zap Request
 class ZapRequest extends Nip01Event {
-  static const int KIND = 9734;
+  static const kZapRequestKind = 9734;
```

Comment:
maybe we need to rethink the kind naming:
kindZapRequestKind is dublicated
Proposal:

either keep short kTextNote or kindTextNote everywhere, also in definition class =>
Nip01Event.kTextNote
Nip01Event.kindTextNote

instead of  
Nip01Event.kKind